dask_ml/model_selection/_search.py

```diff
@@ -55,6 +55,9 @@
 )
 from .utils import DeprecationDict, is_dask_collection, to_indexable, to_keys, unzip
 
+if SK_024:
+    from sklearn.base import _is_pairwise
```

Comment:
Do you know if there's a non-private way to check this? cc @thomasjpfan in case you know off the top of your head.

Regardless, I think this would need an `else` branch defining it as the old way (`estimator._get_tags().get("_pairwise", False)`.

Reply:
@TomAugspurger not sure if there is a non-private way to check it.

I have the `else` branch here: 
https://github.com/abduhbm/dask-ml/blob/62951ab909a0128eda0a67db728ee4062f6fbedf/dask_ml/model_selection/_search.py#L203-L206

Reply:
If `estimator._get_tags().get("_pairwise", False)` has been working for you, then you can use `"pairwise"` as the key instead.

`_is_pairwise` is used to help with the migration from the attribute `_pairwise` to the tag `pairwise`. It's behavior is defined in its docstring: https://github.com/scikit-learn/scikit-learn/blob/2218ec46227c92301ac6837c4a8ae9b8dc5d3960/sklearn/base.py#L819-L828

Reply:
In `test_grid_search_precomputed_kernel` https://github.com/dask/dask-ml/blob/976761ffcd89a6d370b5a29e61d7e80dd5bdd317/tests/model_selection/dask_searchcv/test_model_selection_sklearn.py#L455 only the `_pairwise` attribute is present and it is not False which leads to raising a deprecation warning while running the test.